addons/project_todo/static/src/components/todo_user_ids/todo_user_ids.js

```diff
@@ -0,0 +1,29 @@
+/** @odoo-module */
+
+import { usePopover } from "@web/core/popover/popover_hook";
+import {
+    Many2ManyTagsAvatarFieldPopover,
+    Many2ManyTagsAvatarField,
+} from "@web/views/fields/many2many_tags_avatar/many2many_tags_avatar_field";
+import { _t } from "@web/core/l10n/translation";
+
+export class TodoUserIds extends Many2ManyTagsAvatarField {
+static template = "todo.TodoMany2ManyTagsAvatarUserField";
+    setup() {
+        super.setup();
+        this.placeholder = _t("Asssignees");
```

Comment:
```suggestion
        this.placeholder = _t("Asssignees");
```
directly add in the input/field.